packages/react/src/variables.scss

```diff
@@ -27,6 +30,15 @@ $texteditor-edit-color: $texteditor-color;
 $list-padding-left: $material-list-item-horizontal-padding;
 $footer-height: 70px;
 $footer-left-padding: math.div(($side-panel-min-width - $material-base-icon-size), 2);
+$tooltip-danger-button-color: button_colors.$button-danger-text-hover-bg;
+$tooltip-success-button-color: button_colors.$button-success-text-hover-bg;
+
+$theme-padding: $material-button-horizontal-padding;
+// WA for no button-height in generic theme, remove when design-system is released
+$button-height: calc(26px + ($theme-padding - 10px)/3);
+
+
+$scheduler-toolbar-height: calc($button-height + 2 * $theme-padding);
```

Comment:
We need to discuss the solution. Could we use default height?

Reply:
moved 2nd + operand to new variable